verifier/trustpolicy/trustpolicy.go

```diff
@@ -136,185 +134,16 @@ var (
 
 var supportedPolicyVersions = []string{"1.0"}
```

Comment:
Let's split this into two separate variables and move them to blob.go, oci.go files, so that we don't accidentally introduce regression w.r.t. versioning.